README.md

```diff
@@ -34,146 +38,106 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
+$ vagrant up
+$ export KUBECONFIG="$(pwd)/kubeconfig"
+$ kubectl config use-context vagrant-multi
+```
 
-   ```bash
-   ./rook filesystem unmount --path /tmp/rookFS
-   ```
+Then wait for the cluster to come up and verify that kubernetes is done initializing (be patient, it takes a bit):
 
-6. Cleanup the shared file system from the cluster (this **does** delete the data from the cluster)
+```
+$ kubectl cluster-info
+```
 
-   ```
-   ./rook filesystem delete --name testFS
-   ```
+If you see a url response, you are ready to go.
 
-#### Object Storage
-1. Create an object storage instance in the cluster
+#### Deploy Rook
 
-   ```bash
-   ./rook object create
-   ```
+Rook can be setup and deployed in Kubernetes by simply deploying the [rook-operator](demo/kubernetes/rook-operator.yaml) deployment manifest.
+You will find this manifest and all our example manifest files in the [demo/kubernetes](demo/kubernetes) folder.
 
-2. Create an object storage user
+```
+$ cd demo/kubernetes
+$ kubectl create namespace rook
+$ kubectl create -f rook-operator.yaml
+```
 
-   ```bash
-   ./rook object user create rook-user "A rook rgw User"
-   ```
+Use `kubectl` to list pods in the rook namespace. You should be able to see the following: 
 
-3. Get the connection information for accessing object storage
+```
+$ kubectl -n rook get pod
+NAME                            READY     STATUS    RESTARTS   AGE
+mon0                            1/1       Running   0          1m
+mon1                            1/1       Running   0          1m
+mon2                            1/1       Running   0          1m
+osd-n1sm3                       1/1       Running   0          1m
+osd-pb0sh                       1/1       Running   0          1m
+osd-rth3q                       1/1       Running   0          1m
+rgw-1785797224-9xb4r            1/1       Running   0          1m
+rgw-1785797224-vbg8d            1/1       Running   0          1m
+rook-api-4184191414-l0wmw       1/1       Running   0          1m
+rook-operator-349747813-c3dmm   1/1       Running   0          1m
+```
 
-   ```bash
-   eval $(./rook object connection rook-user --format env-var)
-   ```
+#### Provision Storage
+Before Rook can start provisioning storage, a StorageClass needs to be created. This is used to specify the storage privisioner, parameters, admin secret and other information needed for Kubernetes to interoperate with Rook for provisioning persistent volumes.
+Rook already creates a default admin and demo user, whose secrets are already specified in the sample [rook-storageclass.yaml](demo/kubernetes/rook-storageclass.yaml).
 
-4. Use an S3 compatible client to create a bucket in the object store
+However, before we proceed, we need to specify the Ceph monitor endpoints. You can find them by running this line (you will need `jq`). This will generated a comma-separated list of monitor IPs and ports `6790` and place it into the `monitors` param of the `rook-storageclass.yaml`
```

Comment:
what if you just simplify this whole paragraph to be:
Insert the list of monitors endpoints into the storage class yaml.

For simplicity of the walkthrough, we don't have to explain every step. They can figure it out later if they want.